modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/ManagementToolbar.js

```diff
@@ -50,8 +50,10 @@ function ManagementToolbar({
 	onCreationMenuItemClick = () => {},
 	onInfoButtonClick = () => {},
 	onFilterDropdownItemClick = () => {},
+	onOrderDropdownItemClick = () => {},
```

Comment:
We can create a `noop` function and use it as the default value of all of this methods.

```
const noop = () => {}
```